docs/lib/datastore/fragments/ios/getting-started/40_codegen.md

```diff
@@ -1,11 +1,11 @@
-On iOS the aforementioned `amplify-app` offers an Xcode integration that automatically adds the Amplify-specific files to your project.
+On iOS the `amplify` CLI offers an Xcode integration that automatically adds the Amplify-specific files to your project.
```

Comment:
```suggestion
The `amplify` CLI is integrated with Xcode and will automatically add Amplify-specific files to your project.
```

Reply:
I wouldn't change the wording here as unfortunately the CLI is not exactly "integrated" with Xcode (at least not as integrated as it currently is with Android Studio) :)